sample-notificator-app/src/main/resources/notification-template/email-update-template.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="UTF-8">
+  <title>email-update-template</title>
+</head>
+<body>
+<div id="email-body">
+  <div id="introduction">Dear <span id="person-information">Max Mustermann</span></div>
+  <br>
+  <div id="content">this is an automated email to inform you about recent updates on your subscribed
```

Comment:
you could wrap the text in a `p` tag so that it acts as paragraph. You then can discard the manual line breakage `<br>` which delegates the spacing and so on to the HTML styling rather than the HTML code.

Reply:
Good idea! It's done